### DISCUSSION TITLE
$3

### DISCUSSION THREAD

**Discussion by pan-zhuo** [crystalyragui/MARC2RDA#353] [General]
A few options to map $3 @gerontakos and I talked about in #349 

1) Reify statements as metadata works and apply the option at [Recording a scope for validity of metadata](https://access.rdatoolkit.org/en-US_ala-cfa18e03-17f2-378a-874c-86515bf7e0ac/section_kjz_vtd_ydb)

Example: << [Manifestation URI] rdam:P30305 [$a value] >> rdaw:P10403 [$3 value] .
Problem: complexity; obscure syntax

2) Structured notes with an SES

Example: [Manifestation URI] rdam:P30305 { [$a value] ("According to " [$b value])". Applies to: " [$3 value]. } .
Problem: not compatible with controlled terms/identifiers/IRIs

3) Create URIs as triple subjects for parts of manifestations, aggregated expressions/works

Problem: too many URIs

**Comment by GordonDunsire** [comment]
It is not feasible to identify $3 as a distinct expression that is aggregated in the manifestation being described, or as a part of a multi-unit manifestation. The manifestation is originally described as a whole. Generally, it is fine to apply repeat values of 'materials specified' to the manifestation as a whole.

For example, a 2-lp set with accompanying booklet has two values of carrier type: "audio disc", and "volume".

M21B 338 Carrier Type examples:

338 ##$aaudio disc$0(uri)http://rdaregistry.info/termList/RDACarrierType/1004
338 ##$asheet$2rdacarrier$3liner notes

It is true to state:

ex:1 rdam:P30001 rdact:1004 .
ex:1 rdam:P30001 "sheet" .
ex:1 rdam:P30001 rdact:1048 . // term string matched in RDA VES

To preserve the $3 information, also map the whole field as a note:

ex:1 rdam:P30137 "Carrier type (sheet) applies to liner notes." .

SES: "Carrier type " + "(" + $a + ")" + " applies to " + $3 + "."

It may be feasible to use a boilerplate SES for $3 in general: "applies to " + $3 ???

**Comment by gerontakos** [comment]
I'm going to pass over the aggregations question. There is another discussion (#354) (thank you, Zhuo) for that. I understand the decisions that emerge from discussion 354 may change what we do with $3. 

I'm mapping the MARC 340 with slightly different rdam:noteOnManifestation text for each subfield paired with a $3. We should decide if our boilerplate should always be for a full field or if we want to tailor the noteOnManifestation to the appropriate subfield.

It seems like we now have a clear approach to $3 for most cases. I'm going to write it our below with a suggestion on how to handle it in the spreadhseets.

<pre>
340##$3teacher edition$lcase binding$2rdatb
<manifestation> rdam:typeOfBinding "case binding" ;
                rdam:noteOnManifestation "type of binding (case binding) applies to teacher edition"
Spreadsheet
MARC 340 $l 
    [no conditions] 
    map to: rdam:typeOfBinding
    transformation note: output 340$1 as xs:string
MARC 340 $3 
    condition1: 340$l
    condition2: not($a | $c | $d | $e | $g | $j | $k | $m | $n | $p)
    map to: rdam:noteOnManifestation 
    transformation note: string value of rdam:noteOnManifestation:
        "Type of binding (" + {340$lvalue} ") applies to " + {340$3value} 
</pre>

And a second example:
<pre>
521 2# $aK-3.$bFollett Library Book Co. $3 Workbook
<manifestation> rdam:intendedAidience "K-3" ;
                rdam:noteOnManifestation "Intended audience (K-3) determined by Follett Library Book Co." ;
                rdam:noteOnManifestation "Intended auience (K-3) applies to Workbook"
Spreadsheet
MARC 521 $a
    [no conditions]
    map to: rdam:intendedAidience
    transformation note: output 521$a value as xs:string
MARC 521 $b
    condition1: 521$a
    map to: rdam:noteOnManifestation
    transformation note: string value of rdam:noteOnManifestation:
        "Intended audience (" + {521$avalue} + ") determined by " + {521$bvalue}
MARC 521 $3
    condition1: 521$a
    map to: rdam:noteOnManifestation
    transformation note: string value of rdam:noteOnManifestation:
        "Intended audience (" + {521$avalue} + ") applies to " + {521$3value}
</pre>

**Comment by pan-zhuo** [comment]
If 521$a is already an unstructured description for rdam:P30305, is there anything inherently wrong with adding $b, $3 to it? Almost all places that allow unstructured descriptions have the option to 'record details or other unstructured information.'
I don't quite favor the prospect of having a bunch of general notes with values starting with element names...

**Comment by gerontakos** [comment]
I think you mean you'd like to see one note ($b + $3) instead of two ($b separate from $3)? I separated them in the interest of having a common way to treat $3 across all fields. I doubt there will be many fields like 521 that have a combined "determined by" and "applies to" note. But we could combine them too, that would be no problem. Perhaps we should discuss at the next meeting.
Notes starting with element names; do you have an alternative in mind for matching the note with the field? It's even worse than you suggest, I believe; it's even something we could put in our stand-up comedy routine for metadata professionals: "Hey, it's worse than you think out there, way worse. I mean I saw a place that started their notes with an element name. Then I went across town and the other folks were starting their notes with a full element/value pair!" (We're the "other folks" of course.)

**Comment by CECSpecialistI** [comment]
I agree that we're approaching a good way to handle $3. I like the idea of combining statements when possible, since a $3 should apply to a whole field whenever it appears.

**Comment by CECSpecialistI** [comment]
@gerontakos @pan-zhuo @GordonDunsire Are you all happy with the results of this discussion as a way to map $3 as it occurs moving forward? Or do we need to model out how we will deal with $3 in structured descriptions? Either way, would anyone be willing to write out some documentation for how we will treat $3 so we have something to link / refer to in the mapping spreadsheets?

**Reply by GordonDunsire** [reply]
I don't see any problems with a combined note (a note is a note). I'm not sure what is meant by '$3 in structured descriptions' - is it 'not as a note'? As I mentioned, I don't think it's possible to distinguish a manifestation sub-unit from an aggregated expression, so we can't determine which structure (aggregation or multiple unit) to use ...

**Reply by CECSpecialistI** [reply]
Sorry, that was unclear--I think the $3 will need to be a note, but sometimes it needs to be a note about/attached in some way to a structured description (for example, when a $3 occurs at the start of a 7XX or 3XX field). I'm not sure how this will work when we aren't transforming the field to a note.

**Reply by gerontakos** [reply]
I'm thinking ... how about we treat $3 field-by-field for now; when we decipher the common patterns, we (Zhuo and I) will write the instructions for handling it sometime in January. Assuming that's ok, I've created an agenda item for our next transformation meeting to make sure this gets done. Let us know if that's seems unwise.

**Reply by CECSpecialistI** [reply]
seems like a good idea to me! thanks @gerontakos
